src/persist/src/postgres.rs

```diff
@@ -370,7 +388,7 @@ impl Consensus for PostgresConsensus {
     async fn truncate(&self, key: &str, seqno: SeqNo) -> Result<usize, ExternalError> {
         let q = "DELETE FROM consensus
                 WHERE shard = $1 AND sequence_number < $2 AND
-                EXISTS(
+                EXISTS (
```

Comment:
I started to add an order by / limit here, and then I discovered Postgres doesn't have `DELETE FROM ... LIMIT`, TIL!

Reply:
also TIL